### DISCUSSION TITLE
Error | No module named 'customtkinter'

### DISCUSSION THREAD

**Discussion by TheDoDoo** [TomSchimansky/CustomTkinter#883] [Q&A] [answered]
Hi,
I wanted to use CustomTkinter for my project but got this error "No module named 'customtkinter'"
![grafik](https://user-images.githubusercontent.com/96340450/208257234-075c371f-ced7-41ba-b3c9-5e2bf6b6184a.png)
![grafik](https://user-images.githubusercontent.com/96340450/208257228-63ffc4fc-93d0-4bbb-9c13-252ccdd7918f.png)

I installed as per the description
![grafik](https://user-images.githubusercontent.com/96340450/208257317-23d5d877-fa15-43df-8995-c7fe22bfac99.png)

if I try it again
![grafik](https://user-images.githubusercontent.com/96340450/208257359-46d9a5e3-7f78-45f5-aa7b-15b09e96e109.png)

**Comment by Den-coded** [comment, marked as answer]
Hello, hopefully this will help.
As I can see you're using pycharm as IDE, and to run your proggram inside pycharm, you need to install package inside the pycharm. To do this you need to to go: file, Settings, Project: yourproject, Python Interpreter, here you will see that you probably using virtual enviroment for python interpreter, here you need to click on a plus above packages, now you will see a search bar, search library you want (in your case customtkinter), and click "Install Package", and it should start working. Or you can just actually change Python Intrepreter from Virtual Inviroment to that you have on your pc and probably you will see packages that you have installed are in packages. (you can actually use any method, it will work any way)
![image](https://user-images.githubusercontent.com/121118871/208720997-db78d166-1912-4862-a7df-07151fa3ea2e.png)
(i have highlighted interpreter i had to switch to on the screenshot).
And that's it, for me it worked, I hope my english wasn't so bad because it's not my native language.

**Reply by TheDoDoo** [reply]
yes that helped thanks <3
